lib/transformers/transformers.js

```diff
@@ -91,13 +93,16 @@ const mapOutReq = (requestOptions, reqId, opts = {}) => {
   const metaHeaders = extractHeaders(headersRegex, requestOptions.headers);
 
   let metaBody = {};
-  const isValidToExtractBody =
-    method === 'POST' &&
-    opts.bodyKeys &&
-    requestOptions.body &&
+
+  const hasJsonContentType =
     requestOptions.headers &&
-    'Content-type' in requestOptions.headers &&
-    requestOptions.headers['Content-type'].includes('application/json');
+    ((requestOptions.headers['Content-type'] && requestOptions.headers['Content-type'].includes('application/json')) ||
```

Comment:
This is becoming less and less manageable. Maybe the lowercasing solution worths